docs/source/NeuralNetworkTheory.rst

```diff
@@ -110,9 +110,11 @@ data.
 .. image:: ~Image/NeuralNetworkbyAndrewNg.png
 (by Andrew Ng, Machine Learning Coursera)
 
-.. image:: ~Image/RedExplicada.png
+.. image:: ~Image/RedExplicadaFinished.png
```

Comment:
could you please change the name for this folder. 
Replace: `~Image` by `img`. Let me know if that is ok for you, thank you!

Reply:
Professor,
No problem!